desdeo_tools/utilities/quality_indicator.py

```diff
@@ -69,10 +60,9 @@ def preference_indicator(reference_front: np.ndarray, front: np.ndarray, ref_poi
         float: The factor by which the approximating front is worse than the reference front with respect to all
         objectives taking into account the reference point given and spesifity.
     """
-    ref_front_asf = SimpleASF(reference_front)
-    front_asf = SimpleASF(front)
-    norm = front_asf(front, reference_point=ref_point) + delta - np.min(ref_front_asf(reference_front, reference_point=ref_point))
-    return epsilon_indicator(reference_front, front)/norm
+    front_asf = SimpleASF(np.ones_like(front))
+    norm = front_asf(front, reference_point=ref_point) + delta - min_asf_value
+    return epsilon_indicator(reference_front, front) / norm
 
 
 def hypervolume_indicator(reference_front: np.ndarray, front: np.ndarray) -> float:
```

Comment:
Hypervolume has no concept of a reference front. It uses a front (many solutions) and a reference point (1 point). You can compare two fronts by calculating their hypervolumes individually using the same reference point.